packages/vue-dot/src/sass/variables.scss

```
@@ -0,0 +1,67 @@
+//globale
+$body-font-family: 'source-sans-pro', sans-serif;
+
+//headings with classes
+$headings: (
+	'h1': (
+	  'size': 4.5rem,
+	  'letter-spacing': -1.5px,
+	  'weight': 700
+	),
+	'h2': (
+	  'size': 3rem,
+	  'letter-spacing': 1.5px,
+	  'weight': 500
+	),
+	'h3': (
+	  'size': 2.125rem,
+	  'letter-spacing': .25px,
+	  'text-transform': uppercase,
+	  'weight': 700
+	),
+	'h4': (
+	  'size': 1.5rem,
+	  'letter-spacing': 0rem,
```

```suggestion
	  'letter-spacing': normal,
```
C'est mieux que de mettre `0` (sans unité d'ailleurs) car ça laisse la possibilité au navigateur de modifier l'espacement pour justifier du texte